Formula/jack.rb

```diff
@@ -86,6 +73,27 @@ def plist
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/jackd --version")
+    source_name = "test_source"
+    sink_name = "test_sink"
+    jackd_cmd = "#{bin}/jackd -X coremidi -d dummy"
+    midi_source_cmd = "#{bin}/jack_midiseq #{source_name} 16000 0 60 8000"
```

Comment:
Since this is used once it shouldn't be a variable.

Reply:
Resolved in 197481a.